lib/widgets/recent_dm_conversations.dart

```diff
@@ -102,7 +139,27 @@ class RecentDmConversationsItem extends StatelessWidget {
               overflow: TextOverflow.ellipsis,
               title))),
           const SizedBox(width: 8),
-          // TODO(#253): Unread count
+          Visibility.maintain(
+            visible: unreadCount > 0,
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(3),
+                color: const Color.fromRGBO(102, 102, 153, 0.15),
+              ),
+              child: Padding(
+                padding: const EdgeInsets.fromLTRB(4, 0, 4, 1),
+                child: Text(
+                  style: const TextStyle(
+                    fontFamily: 'Source Sans 3',
+                    fontSize: 16,
+                    height: (18 / 16),
+                    fontFeatures: [FontFeature.enable('smcp')], // small caps
```

Comment:
It shows up for me in Figma's "dev mode" but I didn't find it otherwise:

<img width="1639" alt="image" src="https://github.com/zulip/zulip-flutter/assets/22248748/3d5caea3-61c0-4352-8916-f0b76982a596">

Reply:
Hmm! I guess I should give "dev mode" a try, then. I got a modal recently advertising it as a beta feature, but hadn't tried it out.